base/multimedia.jl

```diff
@@ -225,6 +225,18 @@ function displayable(m::MIME)
     return false
 end
 
+
+macro display(exs...)
+    blk = Expr(:block)
+    for ex in exs
+        push!(blk.args, :(print($(sprint(show_unquoted,ex)*" = "))))
+        push!(blk.args, :(display(begin value=$(esc(ex)) end)))
+        push!(blk.args, :(println()))
+    end
```

Comment:
This assumes that `display` is going to `STDOUT`, whereas it could be going to a window or to some other stream depending on what has been pushed on the display stack.